[PBFIG-194] Design Guidelines: Typography (#3298)
powerhome/playbook: playbook-website/app/views/guides/design_guidelines/typography_guidelines.md

---
title: Typography Guidelines
description: Typography serves as the backbone of any design system. It’s not just page titles and copy, typography is everywhere, in navigation labels, buttons, captions, form fields and more. 
icon: font-case
---

For our Typography Tokens refer to [Typography](https://playbook.powerapp.cloud/visual_guidelines/typography) in [Tokens & Guidelines](https://playbook.powerapp.cloud/visual_guidelines)

#### Index

[Typography Usage & Best Practices](#Typography-Usage-amp-Best-Practices)
‣ [Typography Styles](#Typography-Styles)
‣ [Creating Visual Hierarchy with Typography](#Creating-Visual-Hierarchy-with-Typography)
‣ [Text Modifiers](#Text-Modifiers)
‣ [Typography Kits](#Typography-Kits)
‣ [Typography in Tables](#Typography-in-Tables)
[Typography Accessibility Standards](#Typography-Accessibility-Standards)

> Typography serves as the backbone of any design system. It’s not just page titles and copy, typography is everywhere, in navigation labels, buttons, captions, form fields and more. It is essential for providing context, guiding users and delivering the core content of an application.

Reviewer: ```suggestion
Typography serves as the backbone of any design system. It’s not just for page titles and copy; typography is everywhere, from navigation labels to buttons, captions, form fields, and more. It is essential for providing context, guiding users, and delivering the core content of an application.
```

Playbook’s typography plays a crucial role in shaping the user experience. As a design system primarily used for business software applications, we know that readability is vital for effortless navigation and easy comprehension. To achieve this Playbook uses a professional, unembellished san-serif front and prioritizes intuitive hierarchy in our typography styles. 

> When swapping out the default Playbook font, we recommend using a simple sans-serif font such as Proxima Nova or Ariel.

Reviewer: ```suggestion
When swapping out the default Playbook font, we recommend using a simple sans-serif font such as Proxima Nova or Arial.
```

<br/>
<br/>
Ensuring readable numbers is so crucial for business operations that Playbook offers tabular number spacing as a global prop within the library. It’s recommended on any numeric values for display on any typographic style.
</p>
<img src="https://github.com/powerhome/playbook/assets/48187916/ec72abe4-02b7-4a41-a2fe-60c13f7762c0" width="325px" alt="Tabular Number Spacing">
</div>

### Typography Kits

Playbook offers a range of ready-to-use typography layouts that prioritize displaying clear and concise information.   
Designed to make the best use of the library’s typography options, layouts offered can be used for date and time formats, contact details, addresses, and label/value groupings. 
Through strategic use of color, font size, and font weight, Playbook establishes strong informational and visual hierarchies, ensuring effective communication and enhanced readability.

<img src="https://github.com/powerhome/playbook/assets/48187916/550ddd79-932f-4245-82c1-09b02de35345" alt="Tabular Number Spacing">

### Typography in Tables

> When working with complex tables, the UI is susceptible to becoming information dense and harder to read. Use the following best practices to create clear, readable tables.

Reviewer: ```suggestion
When working with complex tables, the UI is susceptible to becoming information-dense and harder to read. Use the following best practices to create clear, readable tables.
```

<div class="typography-list">
<ul>
<li> Keep your typography choices simple. Avoid overwhelming the design with excessive options.  </li>
<li> Maintain consistency in column headers and body content throughout to ensure coherence.  </li>
<li> Avoid overly large titles or font sizes; instead, rely more on font weight and color variations to establish visual hierarchy effectively.  </li>
<li> If your content is data heavy, consider utilizing the Detail kit over the Body kit.  </li>
<li> For multi-line cells, make use of styles to create readability. </li>
<ul>
     <li> Emphasize primary content using Title 4 or heavier font weights for better clarity and emphasis. </li>
     <li> Use subcaption, detail or other light weight, smaller font sizes for sub-copy or details. </li>
</ul>
</ul>

</div>

<img style="margin:auto; display: block;" src="https://github.com/powerhome/playbook/assets/48187916/4dbcaace-0ce9-43b2-b6f8-9534f278d2b7" width="750" alt="Example table UI">

## Typography Accessibility Standards

#### Color Accessibility Standards

> Accessibility in typography is critical to ensuring inclusivity in usability, as text is such atn integral part of any interface. 

Reviewer: ```suggestion
Accessibility in typography is critical to ensuring inclusivity in usability, as text is such an integral part of any interface. 
```
